From 6f0cddcd68384262b8b1fb315bd1323de4888ecb Mon Sep 17 00:00:00 2001
From: Kirill Kapranov <kirill.kapranov@compulab.co.il>
Date: Wed, 11 Sep 2019 22:33:42 +0300
Subject: [PATCH 35/36] Enable usb to serial converters

Enable the following USB to serial converters that are necessary polular WWAN
modules such as SIMCOM-7100, SIMCOM-5360, Telit xx910.
Enable the USB to serial converter CP210x that is used as a debug serial on a
range of CL boards.

Signed-off-by: Kirill Kapranov <kirill.kapranov@compulab.co.il>
---
 arch/arm64/configs/ucm-imx8m-mini_defconfig | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/configs/ucm-imx8m-mini_defconfig b/arch/arm64/configs/ucm-imx8m-mini_defconfig
index bb8bc3d23baa..d02d663471bb 100644
--- a/arch/arm64/configs/ucm-imx8m-mini_defconfig
+++ b/arch/arm64/configs/ucm-imx8m-mini_defconfig
@@ -208,6 +208,9 @@ CONFIG_USB_NET_SMSC75XX=m
 CONFIG_USB_NET_SMSC95XX=m
 CONFIG_USB_NET_PLUSB=m
 CONFIG_USB_NET_MCS7830=m
+CONFIG_USB_SERIAL=m
+CONFIG_USB_SERIAL_CP210X=m
+CONFIG_USB_SERIAL_OPTION=m
 CONFIG_BRCMFMAC=m
 CONFIG_HOSTAP=y
 CONFIG_RTL_CARDS=m
-- 
2.11.0

